### PR TITLE
fix(kafka): update serviceName/ add import controller/ add @controlle…

### PR DESCRIPTION
**Pull request by morhag90**
Resolves: amplication/amplication/issues/6678, amplication/amplication/issues/6679

### CODE REVIEW

**Review by overbit — CHANGES_REQUESTED** [review]

File: plugins/broker-kafka/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
   "author": "Ofek Gabay",
   "license": "Apache-2.0",
   "devDependencies": {
-    "@amplication/code-gen-types": "^2.0.4",
+    "@amplication/code-gen-types": "^2.0.5-beta.0",
```
Comment (overbit) [inline plugins/broker-kafka/package.json:16]: Comment to wait the release of `code-gen-types` stable

File: plugins/broker-kafka/src/index.ts [hunk 2]
```diff
@@ -345,6 +355,61 @@ class KafkaPlugin implements AmplicationPlugin {
...
+    const generateKafkaClientOptionsImport = importNames(
+      [builders.identifier("generateKafkaClientOptions")],
+      "./kafka/generateKafkaClientOptions"
+    );
+
+    addImports(
+      template,
+      [generateKafkaClientOptionsImport].filter(
+        (x) => x //remove nulls and undefined
+      ) as namedTypes.ImportDeclaration[]
```
Comment (overbit) [inline plugins/broker-kafka/src/index.ts:374]:
```suggestion
      ) as namedTypes.ImportDeclaration[]
```

Why do we need to cast this?

File: plugins/broker-kafka/src/index.ts [hunk 3]
```diff
@@ -345,6 +355,61 @@ class KafkaPlugin implements AmplicationPlugin {
...
+    );
+
+    const test = builders.memberExpression(
+      builders.identifier("app"),
+      builders.identifier("connectMicroservice")
+    );
```
Comment (overbit) [inline plugins/broker-kafka/src/index.ts:380]: This seems redundant. If it's not of any use, let's remove it.

**Review by overbit — CHANGES_REQUESTED** [review]

File: plugins/broker-kafka/src/index.ts (hunk 2 above)
Comment (morhag90) [reply plugins/broker-kafka/src/index.ts:374]: @overbit This is a copy-paste function :)  I can check it without the casting.

**Review by overbit — DISMISSED** [review]

File: plugins/broker-kafka/package.json
Comment (overbit) [inline plugins/broker-kafka/package.json:1]: @morhag90 can you please update the README with a simple example of how to use this plugin?

**Review by overbit — APPROVED** [review]

**Pull request merged**